src/common.ts

```diff
@@ -102,17 +103,22 @@ export const createAtoms = <
       const resultAtom = get(baseStatusAtom)
       return get(resultAtom)
     },
-    (get, set, action: Action) => {
+    (get, set, action: Action | 'refresh') => {
       const observer = get(observerAtom)
       const refresh = () => {
         const queryClient = getQueryClient(get)
         const observerCache = get(observerCacheAtom)
         observerCache.delete(queryClient)
         set(refreshAtom, (c) => c + 1)
       }
+      if (action === 'refresh') {
+        refresh()
+        return observer.getCurrentResult() as unknown as ActionResult
```

Comment:
Does `return undefined as never` help?

Reply:
It gave me TS error.

Reply:
What's the error? `return undefined as unknown as never` maybe?